tests/scipy_stats_test.py

```diff
@@ -564,6 +564,32 @@ def testIssue972(self):
       lsp_stats.norm.cdf(np.full((4,), np.inf, np.float32)),
       check_dtypes=False)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": jtu.format_test_name_suffix("", [shape], [x_dtype, p_dtype]),
+       "x_dtype": x_dtype,
+       "p_dtype": p_dtype,
+       "shape": shape}
+      for shape in [(2,), (3,), (4,), (5,), (6,), (7,), (1,2), (1, 3), (1,4), (1,5)]
```

Comment:
I think we can reduce the number of shapes here to avoid generating an excessive number of test cases. Maybe `fore shape in [(), (4,), (1, 5)]`